nirc_ehr/resources/queries/ehr/necropsyTasks.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+    taskid,
+    updateTitle AS form,
+    assignedTo,
+    created,
+    createdBy,
+    modified,
+    modifiedBy,
+    qcstate
```

Comment:
Since we're re-purposing qcstates here a little, can you also create a new column here containing a new string that translates the qcstates. Mapping qcstates to something like "Request: Pending" -> "Death Entered: Necropsy Pending" and "Review Required" -> "Necropsy Entered: Review Required".

Then in the .query.xml make these two states yellow like the qcstate. And in the default view show these two columns instead of qcstate.

Reply:
Done!